runtime/interface.go

```diff
@@ -79,11 +80,11 @@ type Interface interface {
 	// RevokeEncodedAccountKey removes a key from an account by index, add returns the encoded key.
 	RevokeEncodedAccountKey(address Address, index int) (publicKey []byte, err error)
 	// AddAccountKey appends a key to an account.
-	AddAccountKey(address Address, publicKey *PublicKey, hashAlgo HashAlgorithm, weight int) (*AccountKey, error)
+	AddAccountKey(address Address, publicKey *stdlib.PublicKey, hashAlgo HashAlgorithm, weight int) (*stdlib.AccountKey, error)
```

Comment:
Do we want to expose `stdlib.*` stuff via the interface? or maybe define an alias in the runtime package (similar to `Address`, `HashAlgorithm`, etc.)

Reply:
Great idea! Added type aliases in d264d5fbf588fa843bf9f9588acc4c93a6523a42